scripts/migrate.js

```diff
@@ -70,7 +71,28 @@ const run = async () => {
     await saveWhatsNewIds(whatsNewFiles);
     const definitionFiles = attributeDefFiles.concat(eventDefFiles);
     const files = await all(docs, toVFile).then((files) =>
-      files.sort((a, b) => a.path.localeCompare(b.path))
+      files
+        .sort(
+          (a, b) =>
+            parseInt(a.data.doc.order || 0, 10) -
+            parseInt(b.data.doc.order || 0, 10)
+        )
+        .sort((a, b) => {
+          const aTopic = last(a.data.topics);
+          const bTopic = last(b.data.topics);
+          const getStartedRegex = /^Get(ting)? started/i;
+          const troubleshootRegex = /^Troubleshoot(ing)?/i;
+          if (aTopic === bTopic) {
+            return 0;
+          }
+          if (getStartedRegex.test(aTopic) || troubleshootRegex.test(bTopic)) {
+            return -1;
+          }
+          if (getStartedRegex.test(bTopic) || troubleshootRegex.test(aTopic)) {
+            return 1;
+          }
+          return 0;
+        })
```

Comment:
```suggestion
          const troubleshootRegex = /^Troubleshoot(ing)?/i;
          
          if (aTopic === bTopic) {
            return 0;
          }
          
          if (getStartedRegex.test(aTopic) || troubleshootRegex.test(bTopic)) {
            return -1;
          }
          
          if (getStartedRegex.test(bTopic) || troubleshootRegex.test(aTopic)) {
            return 1;
          }
          
          return 0;
        })
```

I'm a snob and I like some space 😅

Reply:
We could also use that fancy `switch (true)` pattern here as well :thinking:

Reply:
i do like how much cleaner the `switch` is. will run with that. thanks!